viscy/representation/contrastive.py

```diff
@@ -86,9 +86,11 @@ def __init__(
     def forward(self, x):
         x = self.stem(x)
         embedding = self.encoder(x)
+        embedding_norm = F.normalize(embedding, p=2, dim=1)
```

Comment:
Instead of returning normalized embedding here, it can return the embeddings and let the caller do normalization, which is parameter-free. This allows the caller to have access to both, as normalization is destructive.